src/test/java/io/thinkit/edc/client/connector/ContainerBaseTest.java

```diff
@@ -0,0 +1,12 @@
+package io.thinkit.edc.client.connector;
+
+import org.testcontainers.junit.jupiter.Container;
+
+public abstract class ContainerBaseTest {
```

Comment:
Given that this is not a test, the name should not end with `Test`, it could be called `ContainerTestBase`